content/app/development/configuration/stateless/_index.en.md

```diff
@@ -438,31 +435,31 @@ Videre i eksempelet vil betegnelsen *bruker* være synonymt med en virksomhet re
-## Starte instans fra et stateless skjema
+    `lookup.result` is populated with the value from the lookup, bin this case we just write the query string back to the user.
+    Again `true` is returned to indicate that a davavalue has been changed, and `false`if this is not the case.
+## Starting an instance from a stateless form
 
 {{%notice warning%}}
 
-Dette er helt ny funksjonalitet. Oppsett må gjøres manuelt inntil videre og vil ikke være støttet i Altinn Studio.
+This is brand new functionality. Setup must be completed manually untill further notice and will not be supported in Altinn Studio.
 
-**MERK:** for å benytte denne funksjonaliteten må man versjon >= 4.17.2 av nugetpakkene `Altinn.App.PlatformServices`, `Altinn.App.Common` og `Altinn.App.Api`.
+**Note:** To make use of this functionality you must use version >= 4.17.2 of the nuget packages `Altinn.App.PlatformServices`, `Altinn.App.Common` og `Altinn.App.Api`.
 
 {{%/notice%}}
 
-Fra en tilstandsløs applikasjon har man mulighet til å benytte `InstantiationButton`-komponenten til å starte en instans.
-Enn så lenge støtter vi kun å starte en instans innad i samme applikasjonen som stateless skjema vises i. Det å starte en instans i en annn applikasjon er funksjonalitet som kommer.
+In a stateless application you have the opportunity to use the `InstantiationButton` component to start an instance.
+Currently we only support starting an instance from within the same application the stateless form is displayed in. Starting an instance in another application is functionality we will add in the future.
 
-Det er laget en eksempel applikasjon som er satt opp som en innsynstjeneste hvor sluttbruker kan velge å starte en instans på den aktuelle applikasjonen. Denne kan brukes til inspirasjon for videre utvikling. Applikasjonen med kildekode finnes [her.](https://altinn.studio/repos/ttd/start-from-stateless)
+There is an example application which is set up like an access service where the end user can choose to start an instance in the actual application. It can be used as inspiration for further development. The application with source code can be found [here](https://altinn.studio/repos/ttd/start-from-stateless)
 
-### Instansiere med prefill
+### Instantiation with prefill
 
-Et bruksområde for det å starte en instans fra et stateless view kan være at man først ønsker at appen skal oppføre seg som en innsynstjeneste hvor brukeren blir presentert for gitte data som er aktuell. Fra denne informasjonen kan brukeren velge å agere videre på dataen som listes opp, og da er man over på en vanlig innsendingstjeneste.
+One use case where starting an instance from a stateless view can be that you first want the app to act like an access service where the user is presented some given data. From this information the user can choose to act on the listed data, and then use the service as a normal form submittance service.
```

Comment:
This 'access service' complicates the language needlessly, I think... Since the definition of this type of 'access service' has never been defined outside of being a stateless application, it's a bit unclear what we're trying to say, so I suggest not using the term unless we have to.